osgar/tools/lidarview.py

```diff
@@ -106,6 +106,20 @@ def draw(foreground, pose, scan, poses=[], image=None, callback=None, acc_pts=No
                 pygame.draw.line(foreground, (200, 200, 0), a, b, 1)
 
 
+def get_image(data):
```

Comment:
This function looks like a hack. Not knowing much about the rest of the code, I am sort of suggestions how to improve it.

Reply:
?
Actually this was refactoring as JPEG was red on two places. With "depth" data available on both "pins" it has even more sense. One open important question is if depth data is single array of 2D array and if `msgpack` supports that. Here I would like to do also some "color mapping" experiments with the depth data.